dotnet/src/IntegrationTests/Connectors/Anthropic/AnthropicChatCompletionTests.cs

```diff
@@ -0,0 +1,378 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.Anthropic;
+using xRetry;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SemanticKernel.IntegrationTests.Connectors.Anthropic;
+
+public sealed class AnthropicChatCompletionTests(ITestOutputHelper output) : TestBase(output)
+{
+    [RetryTheory]
```

Comment:
Instead of each InlineData I think you can use in the `Theory/Fact` attributes level (applies for all)

```suggestion
    [RetryTheory(Skip = "Thsis test is for....")]
```

Reply:
But then you will be able to enable all providers anthropic, vertex and aws or any of them. I understand correctly?

Reply:
You are currently skipping all, so instead of repeating the same property, just use in the annotation if we need some non-skippable granularity, we can do per `InlineData`